test/connectors/OAuthConnectorSpec.scala

```diff
@@ -0,0 +1,177 @@
+package connectors
+
+import com.typesafe.config.ConfigFactory
+import org.scalatest.matchers.should.Matchers.convertToAnyShouldWrapper
+import org.scalatest.wordspec.AnyWordSpec
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import play.api.http.Status.OK
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.json.Json
+import play.api.test.Helpers.{await, defaultAwaitTimeout}
+import play.api.{Application, Configuration}
+import uk.gov.hmrc.helptosavetestadminfrontend.connectors.OAuthConnector
+import uk.gov.hmrc.helptosavetestadminfrontend.models.AccessToken
+import uk.gov.hmrc.http.test.WireMockSupport
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
+import util.WireMockMethods
+
+import java.util.UUID
+import scala.concurrent.ExecutionContext
+
+class OAuthConnectorSpec extends AnyWordSpec with WireMockMethods with WireMockSupport with GuiceOneAppPerSuite {
+
+  val (desBearerToken, desEnvironment) = "token" -> "environment"
+
+
+  private val config = Configuration(
+    ConfigFactory.parseString(
+      s"""
+         |oauth-access-token-url = "http://$wireMockHost:$wireMockPort"
+         |microservice {
+         |  services {
+         |    oauth-frontend {
+         |      url: "http://localhost:9610"
```

Comment:
Can these 2 services also be injected with wiremock host/port? 

If they are not being interacted with, we can remove them from here. However, if they are, might be worth adding test cases for those interactions aswell and use wiremock host/port here.